<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileItem</class>
 <widget class="QWidget" name="FileItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>40</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="CheckStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="FileName">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="FileType">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="FileSize">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="FilePath">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
